app/src/components/side-bar/chats/ChatsSideBar.tsx

```diff
@@ -11,27 +12,31 @@ interface ChatsSideBarProps {
 const StyledChatsSideBar = styled.div`
   height: 100vh;
   color: var(--color-icon-secondary);
-
   background-color: var(--color-background);
   overflow-y: auto;
+  overflow-x: hidden;
   position: relative;
 `;
+const StyledButtonContainer = styled.div`
+  position: sticky;
+  bottom: 1rem;
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: end;
+  flex-direction: column-reverse;
+  gap: 4rem;
+`;
 function ChatsSideBar({ children }: ChatsSideBarProps) {
   return (
     <StyledChatsSideBar>
       <ChatsSidebarHeader />
+      <StoryListContainer />
```

Comment:
Story list is covering search bar, styling needs to be editied

Reply:
Sure